test/05-OpenURLServer.js

```diff
@@ -28,6 +40,15 @@ const tests = [
       ['metadata.svc.pickupLocation', 'loc123'],
     ],
   },
+  {
```

Comment:
A picky reviewer would not that this test does not exercise `reqIdToUpper`; but I am not such a reviewer :-)

Reply:
>  I like how straightforwardly it all reads.

Thank you, this is my favourite compliment as "straightforward" is what I strive for.